projects/sunbird-player-sdk/src/lib/player-utils/service/error/error.service.spec.ts

```diff
@@ -1,9 +1,15 @@
 import { TestBed } from '@angular/core/testing';
+import { PLAYER_CONFIG } from '../../../sunbird-player-sdk.module';
 import { errorCode, errorMessage } from './../../enums/exceptionLogs';
 import { ErrorService } from './error.service';
 
 describe('ErrorService', () => {
-  beforeEach(() => TestBed.configureTestingModule({}));
+  beforeEach(() => TestBed.configureTestingModule({
```

Comment:
Add test case for both default and override contentCompatibilityLevel with assetions

Reply:
@HarishGangula Updated the test cases, please review the changes.